site/en/docs/webstore/cws-dashboard-listing/index.md

```diff
@@ -27,8 +27,32 @@ your item's store listing:
 - The **primary category** where your item should be listed.
 - Your item's **language** to help users find it.
 
-See [Supplying Images][1] for help on designing the images for your item, and [Branding
-Guidelines][2] for information on how you can use Google brands.
+## Learning more about images and branding
 
-[1]: /docs/webstore/images
-[2]: /docs/webstore/branding
+See [Supplying Images][cws-images] for help on designing the images for your item, and [Branding
+Guidelines][cws-branding] for information on how you can use Google brands.
+
+## Displaying an official URL
```

Comment:
I talked with David, and suggest that we tackle the feature as "Verifying your publisher identity", rather than displaying an official URL. So please update the title, as well as some of the words later, accordingly.

Reply:
Updated per discussions. PTAL.